src/main/webapp/college-info.html

```diff
@@ -7,14 +7,31 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <input type="text" id="school-search"> 
-    <button onclick="getSchoolInfo()">Search</button>
-    <h1 id="school-name"></h1>
-    <h3 id="school-desc"></h3>
-    <div id="students"></div>
-    <div id="race-piechart" style="width: 900px; height: 500px;"></div>
-    <div id="gender-piechart" style="width: 900px; height: 500px;"></div>
-    <div id="admissions"></div>
-    <div id="cost"></div>
+    <header>
+      <h1>Black Student Insight</h1>
+      <input type="text" id="school-search"> 
+      <search-button onclick="getSchoolInfo()">Search</search-button> 
+    </header>
+    <main>
+      <div class="school-header">
```

Comment:
I believe you can handle all of the styling on this header without this div. Correct me if I'm wrong, but if you just removed it and moved all of the .school-header styles into #school-name, I think you'd be pretty good to go.